enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -158,6 +158,12 @@ class Meta:
             'it disappear from all frontends, effectively soft-deleting it. Default is False (deactivated).'
         ),
     )
+    browse_and_request = models.BooleanField(
+        default=False,
+        help_text=(
+            'If selected, it means browse and request is active on this policy.'
+        ),
+    )
```

Comment:
nit: Can we rename it to 'bnr_enabled'? This makes it more specific and clearly defines the purpose of the flag."